ui/src/dataLoaders/components/lineProtocolWizard/verify/LineProtocolVerifyStep.tsx

```diff
@@ -22,21 +21,24 @@ export class VerifyLineProtocolStep extends PureComponent<Props> {
     const {onDecrementCurrentStepIndex, onExit} = this.props
 
     return (
-      <div className="onboarding-step">
-        <Form onSubmit={onExit}>
-          <div className="wizard-step--scroll-area">
-            <FancyScrollbar autoHide={false}>
-              <div className="wizard-step--scroll-content">
-                <StatusIndicator />
-              </div>
-            </FancyScrollbar>
-          </div>
-          <OnboardingButtons
-            onClickBack={onDecrementCurrentStepIndex}
-            nextButtonText="Finish"
-          />
-        </Form>
-      </div>
+      <Form onSubmit={onExit}>
+        <Overlay.Body style={{textAlign: 'center'}}>
+          <StatusIndicator />
+          <p>
```

Comment:
It might be overkill, but it could also be worth keeping this as a separate component since we're using it in two different places:

https://github.com/influxdata/influxdb/pull/16276/files#diff-52a6ec5c0192e8f9072fe553ed255ac3R27-R35
&
https://github.com/influxdata/influxdb/pull/16276/files#diff-e52ed602ead645dcab7e5c9042d9bfd1R62-R70

Reply:
I'm gonna do it!